libtelemetry/src/main/java/com/mapbox/android/telemetry/LocationEvent.java

```diff
@@ -38,6 +38,11 @@ public class LocationEvent extends Event implements Parcelable {
   @SerializedName("permissionStatus")
   private final String permissionStatus;
 
+  public LocationEvent(String sessionId, double latitude, double longitude, long timestampUtcMs,
```

Comment:
should we also mark this as deprecated and use the the one with permissionStatus by default?

Reply:
@pengdev I was thinking about that. Will it cause compilation warnings/errors for customers?

Reply:
no, it will be a lint warning.

Reply:
done